content/working_with/manager/update-deployment.md

```diff
@@ -6,39 +6,61 @@ weight: 650
+    - the updated node properties
+    - the updated node instance relationships
+1. The `uninstall` operations are executed on each deleted node instance.
+1. The `install` operations are executed on each added node instance.
+1. The `check_status` (and `heal` if necessary) operations are executed on all started node instances in the deployment.
+1. The `check_drift` (and `update` if necessary) operations are executed on all started and healthy node instances in the deployment.
+1. The `reinstall` operations are executed on each changed node instance.
+1. The deleted nodes and node-instances are removed from the {{< param cfy_manager_name >}} data storage.
 
-1. The steps composing the deployment update are extracted.
-2. All the 'added' and 'modified' changes are updated in the data model.
-3. The `update` workflow is executed. As a part of the (default) update workflow execution:
-    - The `unlink` operation is executed in regard to each removed relationship.
-    - The `uninstall` workflow is executed on each of the removed nodes.
-    - Old plugins that were outdated are being uninstalled.
-    - The `install` workflow is executed on each of the added nodes.
-    - The `establish` operation is executed in regard to each added relationship.
-    - New plugins that were updated are being installed.
-    - The `reinstall` workflow (uninstall and install) is executed on each of the modified nodes and each of the nodes that were explicitly marked for reinstall.
-4. All 'removed' changes are updated in the data model.
+For a more in-depth description of these steps, see the [workflow documentation]({{<relref "working_with/workflows/built-in-workflows#the-update-workflow">}})
 
-**Workflow/operation execution during a deployment update**<br>
-Stage 3 of the deployment update flow comprises only the cases in which a workflow or an operation is executed during a deployment update. That is, when changing description, removing a workflow, modifying the `install-agent` property or any other step that is not add/remove/modify node or relationship, no workflow or operation is executed. Note that since version 4.4, modifying an existing node (changing its properties and/or operations) will cause automatic reinstallation of this node, unless the flag `--skip-reinstall` has been supplied.
+For notes about implementing the `check_drift` and `update` operations, see [implementing drift check]({{<relref "working_with/manager/update-deployment#implementing-drift-check">}})
 
 ## Using {{< param cfy_console_name >}} to Update a Deployment
-To update deployment from the {{< param cfy_console_name >}} go to [Services page]({{< relref 
-"working_with/console/pages/services-page.md" >}}), select deployment from the left pane, then click on the 
+To update deployment from the {{< param cfy_console_name >}} go to [Services page]({{<relref "working_with/console/pages/services-page.md">}}), select deployment from the left pane, then click on the
```

Comment:
```suggestion
To update a deployment from the {{< param cfy_console_name >}} go to [Services page]({{<relref "working_with/console/pages/services-page.md">}}), select **deployment** from the left pane, then click on the
```
The next line should be following here without <enter>

Reply:
line breaks in the source .md are not rendered in the resulting html

(also btw I note that I only moved this line, didn't actually change it)